__tests__/app.test.js

```diff
@@ -74,3 +74,40 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("200: GET responds with a single article object", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          author: "icellusedkars",
+          title: "Eight pug gifs that remind me of mitch",
```

Comment:
Let's check for a generic match with expect.objectContaining or .toMatchObject rather than comparing directly! just incase the article changes in the future and breaks the test! 

The only thing we can test that won't change is the review id is 3 :)